src/test/serializable/base.py

```diff
@@ -0,0 +1,33 @@
+from ja.common.message.base import Serializable
+from typing import Dict
+from unittest import TestCase
+
+
+class SimpleSerializable(Serializable):
+    def __init__(self, x: int) -> None:
+        self.x = x
+
+    def to_dict(self) -> Dict[str, object]:
+        return {'x': self.x}
+
+    @classmethod
+    def from_dict(cls, property_dict: Dict[str, object]) -> "SimpleSerializable":
+        assert len(property_dict) == 1
+        assert 'x' in property_dict
+        assert isinstance(property_dict['x'], int)
```

Comment:
For a trivial unit test like this I'd say it doesn't matter but for more complicated unit tests please use `self.assertTrue` with a meaningful message to make debugging easier.

Reply:
ah right, I'll use it.

Reply:
This is not a `TestCase`, rather a dummy implementation.